observer.cpp

```diff
@@ -87,5 +87,28 @@ void test_observer()
         assert(o.get_params() == params);
     }
 #endif
+
+//#define FIX_ISSUE_81
+#ifdef FIX_ISSUE_81
```

Comment:
This is a good starting framework for testing storage, although it needs som tweaks.
But issue #81 has a slightly larger scope, where the saving as and loading need to be tested as well. Have a look in other tests for saving and loading how i do it.